packages/resizable-mixin/resizable-mixin.ts

```diff
@@ -0,0 +1,68 @@
+import { LitElement } from 'lit-element';
+import getResizeObserver from './resize-observer-polyfill';
+import { ResizableClass } from './resizable-class';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Constructor<T = object> = new (...args: any[]) => T;
+
+const resizables = new Set();
```

Comment:
Is this double bookkeeping necessary? I see it's used in disconnectedCallback to make sure all elements are unobserved when the set is empty. Shouldn't observe/unobserve logic in connected/disconnected callbacks be enough to make sure there are no unnecessary registrations?

Reply:
Removed. I agree that it's not really necessary and just adds extra logic.